gbus/abstractions.go

```diff
@@ -109,6 +112,12 @@ type HandlerRegister interface {
 //MessageHandler signature for all command handlers
 type MessageHandler func(invocation Invocation, message *BusMessage) error
 
+func (mg MessageHandler) Name() string {
```

Comment:
doesn't feel right that implementation is defined  the abstractions.go file.
Maybe extract both the MessageHandler and the Name() method to a message_handler.go file ?